crates/core/tedge/src/main.rs

```diff
@@ -31,7 +31,6 @@ use tracing::log;
 static ALLOCATOR: Cap<alloc::System> = Cap::new(alloc::System, usize::MAX);
 
 fn main() -> anyhow::Result<()> {
-    clap_complete::CompleteEnv::with_factory(TEdgeCli::command).complete();
```

Comment:
Oops, must've accidentally deleted this when removing the CryptoProvider stuff.

I've created #3373 to address the fact this went undetected.